packages/hooks/src/index.ts

```diff
@@ -90,7 +90,7 @@ export {
   formatSocial,
   isSameAddress,
   removeEmptyFromObject,
-  parsePrice,
```

Comment:
please keep `parsePrice` to not add any breaking change even if it not used internally